notebooks/lightning_segmentation.ipynb

```diff
@@ -0,0 +1,965 @@
+{
```

Comment:
<p>Add some text here with some information/exercises :) :</p><ul><li>If gpu is None and the notebook is run in a colab environment remind the user to activate a GPU environment</li><li>Exercise for the user: switch from CPU transforms to GPU transforms (nice for Colab) </li><li>Add example output of this cell in a markdown below (e.g. output for a few epochs)</li><li>Report training time on cpu and gpu in colab environment </li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PhoenixDL/rising/pull/74/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>